doc/Quick_Start/README.md

```diff
@@ -66,21 +66,25 @@ https://docs.rakwireless.com/Product-Categories/WisBlock/RAK5005-O/Datasheet/#wi
 
 The RAK5005-O board offers several GPIO's on solder pads or on the WisBlock Sensor or WisBlock IO modules. These GPIO's are named IO1 to IO6 and SW1. These GPIO's are connected to GPIO's of the RAK4631 module.    
 **RAK5005-O GPIO mapping to RAK4631 GPIO ports**
```

Comment:
It would look better if there's a single space before this header

Reply:
Changed.